package.json

```diff
@@ -365,8 +365,7 @@
 	},
 	"extensionDependencies": [
 		"ms-kubernetes-tools.vscode-kubernetes-tools",
-		"redhat.vscode-yaml",
-		"redhat.vscode-commons"
+		"redhat.vscode-yaml"
 	],
 	"dependencies": {
 		"@redhat-developer/vscode-redhat-telemetry": "^0.1.1",
```

Comment:
^0.2.0

Reply:
I was trying to keep the change minimal to demonstrate it. But updated and same error https://app.circleci.com/pipelines/github/camel-tooling/vscode-camelk/1069/workflows/06ba6b81-9666-487c-8bf1-3c3997b9b51e/jobs/1359?invite=true#step-110-452